app/views/users/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>All Users</h1>
\ No newline at end of file
+<% @users.each do |user| %>
+  <div class='main-container'>
+    <div class='user-container'>
+      <img src="<%= user.photo %>" class='user-image'>
+    </div>
+    <h2 class='users-info'>
+      <%= link_to user.name, user_path(user) %>
+    </h2>
+    <p class='post-num'>Number of posts: <%= user.posts_counter %></p>
+  </div>
+<% end %>
```

Comment:
- [ ] it would be better to add spacing margin and styles to put the elements as in the wireframe 
- the profile picture is bigger than expected and not well positioned as the wireframe it should be in the same line with users' info in order to match the wireframe 
- Yours 

<img width="1139" alt="Screenshot 2023-06-02 at 11 22 23" src="https://github.com/lugard1/blogapp/assets/61790587/ffa12972-5d3c-4779-85ae-ea19966c8559">

- Design 
<img width="459" alt="Screenshot 2023-06-02 at 11 26 36" src="https://github.com/lugard1/blogapp/assets/61790587/be5fd94f-bd43-4420-95f1-06c91353e137">